addons/mail/models/mail_thread.py

```diff
@@ -2822,20 +2822,20 @@ def _notify_thread_by_inbox(self, message, recipients_data, msg_vals=False, **kw
           access message content in db;
         """
         bus_notifications = []
-        inbox_pids = [r['id'] for r in recipients_data if r['notif'] == 'inbox']
-        if inbox_pids:
+        inbox_recipients = [r for r in recipients_data if r['notif'] == 'inbox']
+        if inbox_recipients:
             notif_create_values = [{
                 'author_id': message.author_id.id,
                 'mail_message_id': message.id,
                 'notification_status': 'sent',
                 'notification_type': 'inbox',
-                'res_partner_id': pid,
-            } for pid in inbox_pids]
+                'res_partner_id': recipient['id'],
+            } for recipient in inbox_recipients]
             self.env['mail.notification'].sudo().create(notif_create_values)
 
             message_format_values = message.message_format()[0]
-            for partner_id in inbox_pids:
-                bus_notifications.append((self.env['res.partner'].browse(partner_id), 'mail.message/inbox', dict(message_format_values)))
+            for recipients in inbox_recipients:
```

Comment:
Why this change ? Remaining of previous implementation ? Does not seem to change ?